src/mats-method-workspace/mats-method-checks.service.spec.ts

```diff
@@ -65,4 +65,19 @@ describe('Mats Method Checks Service Test', () => {
       }
     });
   });
+
+  describe('MATSMTH-5 Checks', () => {
+    it('Should get [MATSMTH-5-A] error', async () => {
+      payload.beginDate = new Date('2023-02-28');
+      payload.beginHour = 5;
+      payload.endDate = new Date('2023-02-28');
+      payload.endHour = 1;
+
+      try {
```

Comment:
This test won't fail even if the validation is not working right and fails to throw an error for bad dates

Reply:
This is just checks beginDate is after endDate or not. This is currently throwing error. I am getting coverage for those lines.

Reply:
![image](https://user-images.githubusercontent.com/102534033/222558350-8d106cda-54bd-4b7a-8094-ca02b4649081.png)